distributed/tests/test_worker.py

```diff
@@ -1634,3 +1634,27 @@ def bad_heartbeat_worker(*args, **kwargs):
                 else:
                     assert w.status == "closed"
     assert "Heartbeat to scheduler failed" in logger.getvalue()
+
+
+@gen_cluster(client=True, nthreads=[("127.0.0.1", 1)])
+async def test_delete_data(client, s, w):
+    df = dask.datasets.timeseries(dtypes={"x": int, "y": float}, freq="1s").reset_index(
+        drop=True
+    )
+
+    await client.compute(df.map_partitions(lambda df: df.__sizeof__()))
```

Comment:
Is everything here necessary?

Reply:
Seems like getting rid of `await client.compute(df.map_partitions(lambda df: df.__sizeof__()))` works just the same, so just the dataframe creation suffices.

Reply:
Removed in latest commits.